### PR TITLE
Add changelog for 2.0

### PR DESCRIPTION
**Pull request by laurit**

### CODE REVIEW

**Review by theletterf — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,31 @@ and this repository adheres to [Semantic Versioning](https://semver.org/spec/v2.
 
 ## Unreleased
 
+The 2.0.0 release contains significant breaking changes that will most likely affect all users,
```
Comment (akubik-splunk) [inline CHANGELOG.md:11]: Use 2.0.0 Alpha  in relation to SPlunk-Otel-Java everyhvere

Comment (akubik-splunk) [inline CHANGELOG.md:11]: Use 2.0.0 Alpha  in relation to SPlunk-Otel-Java everyhvere

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,31 @@ and this repository adheres to [Semantic Versioning](https://semver.org/spec/v2.
...
+
+### General
+
+- OpenTelemetry Java SDK has been updated to version 1.34.1.
+- OpenTelemetry Instrumentation for Java has been updated to version 2.0.0.
+
+### Breaking Changes
+- Micrometer based metrics were removed in favour of OpenTelemetry based metrics.
+- The default OTLP protocol has been changed from `grpc` to `http/protobuf` in order to align with
+  the [specification](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.28.0/specification/protocol/exporter.md#specify-protocol).
```
Comment (akubik-splunk) [inline CHANGELOG.md:24]: Capitalize

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,6 +8,31 @@ and this repository adheres to [Semantic Versioning](https://semver.org/spec/v2.
...
+  You can switch to the `grpc` protocol using `OTEL_EXPORTER_OTLP_PROTOCOL=grpc`
+- The OTLP metrics exporter is now enabled by default. You can disable it using
+  `OTEL_METRICS_EXPORTER=none` or `-Dotel.metrics.exporter=none`.
+- The OTLP logs exporter is now enabled by default. You can disable it using
+  `OTEL_LOGS_EXPORTER=none` or `-Dotel.logs.exporter=none`.
+- splunk-otel-javaagent-all.jar is not published anymore, use splunk-otel-javaagent.jar (##1567)
```
Comment (breedx-splk) [inline CHANGELOG.md:30]:
```suggestion
- splunk-otel-javaagent-all.jar is not published anymore, use splunk-otel-javaagent.jar (#1567)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -8,6 +8,31 @@ and this repository adheres to [Semantic Versioning](https://semver.org/spec/v2.
...
+please be sure to read the breaking changes below carefully.
+
+Note: 2.0.0 release is considered experimental, Splunk recommends using 1.x version of the agent.
+
+### General
+
+- OpenTelemetry Java SDK has been updated to version 1.34.1.
+- OpenTelemetry Instrumentation for Java has been updated to version 2.0.0.
+
+### Breaking Changes
```
Comment (breedx-splk) [inline CHANGELOG.md:21]:
```suggestion
### ⚠️⚠️ Breaking Changes
```

**Review by breedx-splk — APPROVED** [review]
LGTM just a few comments, and probably worth adding in "Alpha" as @akubik-splunk suggested. 🤘🏻

**Pull request merged**